application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -56,4 +59,27 @@ public static String escapeMarkdown(String text) {
         return afterEscape.replace("\\```", "\\`\\`\\`");
     }
 
+    /**
+     * Converts a slash command text to a mentioned slash command, which you can directly click on
+     * in Discord
+     *
+     * @param guild guild that owns the command
+     * @param command command name
+     * @return Formatted string for mentioned slash command
+     * @throws IllegalArgumentException when the command name doesn't match with the guild's
+     *         commands
+     */
+    public static String mentionSlashCommand(Guild guild, String command) {
+        AtomicLong commandId = new AtomicLong();
+        guild.retrieveCommands()
+            .queue(commands -> commands.stream()
+                .filter(c -> c.getName().equalsIgnoreCase(command))
+                .findFirst()
+                .ifPresentOrElse(c -> commandId.set(c.getIdLong()), () -> {
+                    throw new IllegalArgumentException("Command %s does not exist in guild %s"
+                        .formatted(command, guild.getId()));
+                }));
```

Comment:
This would be more readable and idiomatic if split into:
```java
.map(c -> commandId.set(...))
.orElseThrow(() -> new IllegalArgumentException(...))
```